gputop/remote/gputop.js

```diff
@@ -79,7 +78,7 @@ function on_jquery_ready() {
         ProtoBuf.protoFromString(proto, proto_builder, "gputop.proto");
         gputop = new Gputop();
         gputop_ready(gputop);
-    });
+    }, function (error) { console.log(error); });
```

Comment:
I wonder if we should make http_get have a default error handler that logs an error like this, to save duplicating this kind of minimal logging handler